docker-compose.test.yml

```
@@ -22,16 +30,21 @@
       - '9324:9324'
     environment:
       - NEXT_PUBLIC_PRODUCTION_API_URL=http://prod:3001
+    healthcheck:
+        <<: *default-healthcheck
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 8

(indentation)